handler/capture/capture_test.go

```diff
@@ -0,0 +1,82 @@
+package capture
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/phemmer/sawmill"
+	"github.com/phemmer/sawmill/event"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Handler_iface(t *testing.T) {
```

Comment:
Nit, but can we use the standard namding convention for these? Meaning no underscore after `Test`. Underscore is fine later on if it's a different way of testing that specific function/scenario/whatever